VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -609,6 +616,49 @@ public Properties trackUserFindsCourses() {
         return null;
     }
 
+    /**
+     * This function is used to track if user clicks on Create Account on registration screen
+     * @return
+     */
+    @Override
+    public Properties trackCreateAccountClicked() {
+        try{
+            VideoEvent vEvent = new VideoEvent();
```

Comment:
I have renamed the class to SegmentAnalyticsEvent. 

@rohan-dhamal-clarice  - Please review